libfreerdp/core/redirection.c

```diff
@@ -384,15 +385,15 @@ static BOOL rdp_redirection_read_target_cert(rdpRedirection* redirection, const
 				}
 				break;
 			default: /* ignore unknown fields */
-				WLog_WARN(TAG,
-				          "Unknown TargetCertificate field type %" PRIu32 ", encoding %" PRIu32
-				          " of length %" PRIu32,
-				          type, encoding, plength);
+				WLog_VRB(TAG,
```

Comment:
`trace` might be a bit too hidden, maybe `debug` instead?

Reply:
Verbose is fine, we're not interested by most of the exported items (I'm even wondering if there should have a log).